edit.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
Why?

Reply:
I set up the button so I'm prepped for my next task 😆